mk/spksrc.download.mk

```diff
@@ -63,8 +63,12 @@ download_target: $(PRE_DOWNLOAD_TARGET)
 	      if [ ! -f $${localFile} ]; then \
 	        rm -fr $${localFolder}.part ; \
 	        echo "git clone $${url}" ; \
-	        git clone --no-checkout --quiet $${url} $${localFolder}.part ; \
+	        git clone --no-checkout --recursive --quiet $${url} $${localFolder}.part ; \
 	        git --git-dir=$${localFolder}.part/.git --work-tree=$${localFolder}.part checkout --quiet $(PKG_GIT_HASH) ; \
+	        echo "git submodule update --init --recursive" ; \
```

Comment:
Doesn't the `clone --recursive` on L66 imply that the submodules are up-to-date? In other words: what's the purpose of L68-L71?